perf/benchmarkapps/GrpcClient/ClientOptions.cs

```diff
@@ -22,7 +22,7 @@ namespace GrpcClient
 {
     public class ClientOptions
     {
-        public string? Url { get; set; }
+        public Uri? Url { get; set; }
```

Comment:
The actual change I wanted to make to figure out why benchmarks were failing.

"Might as well update System.Commandline while I'm here. Should be easy." <- famous last words

😢